src/SmartComponents/BaselinesPage/BaselinesPage.js

```diff
@@ -90,70 +86,43 @@ export class BaselinesPage extends Component {
-                        tableData={ baselineTableData }
-                        loading={ loading }
-                        columns={ columns }
-                        kebab={ true }
-                        createButton={ true }
-                        exportButton={ true }
-                        onClick={ this.fetchBaseline }
-                        onBulkSelect={ this.onBulkSelect }
-                        selectedBaselineIds={ selectedBaselineIds }
-                        totalBaselines={ totalBaselines }
-                        permissions={ permissions }
-                        hasSwitch={ true }
-                        notificationsSwitchError={ notificationsSwitchError }
-                        emptyState={ emptyState }
-                    />
-                </div>
-            </CardBody>
-        );
-    }
 
-    renderEmptyState = (permissions) => {
-        const { baselineError, emptyState, loading, revertBaselineFetch } = this.props;
-        const { errorMessage } = this.state;
-
-        if (!baselineError.status) {
-            return <EmptyStateDisplay
-                icon={ AddCircleOIcon }
-                title={ EMPTY_BASELINES_TITLE }
-                text={ EMPTY_BASELINES_MESSAGE }
-                button={ <CreateBaselineButton
-                    emptyState={ emptyState }
+            <div>
+                <BaselinesTable
+                    tableId='CHECKBOX'
+                    hasMultiSelect={ true }
+                    onSelect={ this.onSelect }
+                    tableData={ baselineTableData }
+                    loading={ loading }
+                    columns={ columns }
+                    kebab={ true }
+                    createButton={ true }
+                    exportButton={ true }
+                    onClick={ this.fetchBaseline }
+                    onBulkSelect={ this.onBulkSelect }
+                    selectedBaselineIds={ selectedBaselineIds }
+                    totalBaselines={ totalBaselines }
                     permissions={ permissions }
-                    loading={ loading } /> }
-            />;
-        } else if (baselineError.status !== 200 && baselineError.status !== undefined) {
-            return <EmptyStateDisplay
-                icon={ ExclamationCircleIcon }
-                color='#c9190b'
-                title={ 'Baselines cannot be displayed' }
-                text={ errorMessage }
-                error={ 'Error ' + baselineError.status + ': ' + baselineError.detail }
-                button={ <a onClick={ () => revertBaselineFetch('CHECKBOX') }>
-                    <UndoIcon className='reload-button' />
-                        Retry
-                </a> }
-            />;
-        }
+                    hasSwitch={ true }
+                    notificationsSwitchError={ notificationsSwitchError }
+                    emptyState={ emptyState }
+                    revertBaselineFetch = { revertBaselineFetch }
```

Comment:
Make sure `revertBaselineFetch`, `baselineError` and `tableId` get passed to `BaselinesTable` from `BaselinesPage`, `AddSystemModal` and `CreateBaselineModal`.